examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    generate_data('sigir-sku', 1000).head()
</pre><p>In the original The file&nbsp;<code>sku_to_content.csv</code>&nbsp;contains also <code>category_hash</code> feature. If you want to remove it may be you can add a note here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Yes, you are right! Well spotted, I was not aware I was missing it here, thank you for pointing it out. I brought it back (added it to `schema.json` for the synthetic data and also added it to the workflow for processing the actual data) but I have removed the `image_vector` and added a note as you advise since I don't believe we will be using it and rather will go with the `description_vector` which lends itself much better to this example (the `image_vector` is of length greater than 68k in the original data)